source/patterns/@aws-solutions-constructs/aws-lambda-opensearch/README.md

```diff
@@ -108,9 +108,9 @@ new LambdaToOpenSearch(this, "sample",
 |:-------------|:----------------|-----------------|
 |existingLambdaObj?|[`lambda.Function`](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_lambda.Function.html)|Existing instance of Lambda Function object, providing both this and `lambdaFunctionProps` will cause an error.|
 |lambdaFunctionProps?|[`lambda.FunctionProps`](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_lambda.FunctionProps.html)|User provided props to override the default props for the Lambda function.|
-|domainProps?|[`opensearchservice.CfnDomainProps`](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_opensearchservice.CfnDomainProps.html)|Optional user provided props to override the default props for the OpenSearch Service.|
-|domainName|`string`|Domain name for the Cognito and the OpenSearch Service.|
-|cognitoDomainName?|`string`|Optional Cognito domain name, if provided it will be used for Cognito domain, and `domainName` will be used for the OpenSearch domain.|
+|openSearchDomainProps?|[`opensearchservice.CfnDomainProps`](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_opensearchservice.CfnDomainProps.html)|Optional user provided props to override the default props for the OpenSearch Service.|
+|openSearchDomainName|`string`|Domain name for the OpenSearch Service.|
+|cognitoDomainName?|`string`|Optional Cognito domain name, if provided it will be used for Cognito domain, and `openSearchDomainName` will be used for the OpenSearch Service domain.|
```

Comment:
Amazon Cognito domain name. If omitted the Cognito domain will default to the OpenSearch Service domain name.